dipy/align/scalespace.py

```diff
@@ -0,0 +1,422 @@
+from dipy.align import floating
+import numpy as np
+import numpy.linalg as npl
+import scipy as sp
+import scipy.ndimage as ndimage
+import scipy.ndimage.filters as filters
+
+
+class ScaleSpace(object):
```

Comment:
This class is to build an ANTS-like scale space. Here, the sigma along each x,y,z direction depends on the voxel size (this SS is anisotropic)